### PR TITLE
Move nomad examples to plugin extend

### PR DESCRIPTION
**Pull request by RubelMozumder**
Closed in favor of #448

### CODE REVIEW

File: src/pynxtools/testing/nomad_test_units.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    """Compare the json file generated from yaml file."""
+    ...
+
+
+def test_nxs_file_is_generated():
+    """Check if nexus file has been created."""
+    ...
+
+
+def test_parse_nxs_file():
```
Comment (RubelMozumder) [inline src/pynxtools/testing/nomad_test_units.py:49]: Comment:
1. Parsing
2. normalization
3. Storage
for nexus file.

File: src/pynxtools/testing/nomad_example.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    Test if NOMAD ExampleUploadEntryPoint works.
+
+    Args:
+        entrypoint (nomad.config.models.plugins.xampleUploadEntryPoint): The entry point to test.
+        plugin_package (str): The plugin package to set on the entry point.
+        expected_local_path (str): The expected local path after loading.
+
+    """
+    setattr(entrypoint, "plugin_package", plugin_package)
+    entrypoint.load()
```
Comment (RubelMozumder) [inline src/pynxtools/testing/nomad_example.py:107]:
```suggestion
    attr = getattr(entrypoint, "plugin_package", None)
    if not attr:
        setattr(entrypoint, "plugin_package", plugin_package)
    entrypoint.load()
```

Comment (RubelMozumder) [reply src/pynxtools/testing/nomad_example.py:107]: The reason to check if the `plugin_packge` attr has already come with `entrypoint`.

**Comment by RubelMozumder** [comment]
Closed in favor of #448

**Comment by lukaspie** [comment]
@RubelMozumder I thought there were some good ideas here, maybe we can come back to these when #448 is done.

**Pull request closed without merging**